eng/targets/Services.props

```diff
@@ -21,5 +21,6 @@
     <ServiceHubService Include="Microsoft.VisualStudio.Razor.TagHelperProvider" ClassName="Microsoft.CodeAnalysis.Remote.Razor.RemoteTagHelperProviderServiceFactory"/>
     <ServiceHubService Include="Microsoft.VisualStudio.Razor.ClientInitialization" ClassName="Microsoft.CodeAnalysis.Remote.Razor.RemoteClientInitializationServiceFactory" />
     <ServiceHubService Include="Microsoft.VisualStudio.Razor.UriPresentation" ClassName="Microsoft.CodeAnalysis.Remote.Razor.RemoteUriPresentationServiceFactory" />
+    <ServiceHubService Include="Microsoft.VisualStudio.Razor.FoldingRange" ClassName="Microsoft.CodeAnalysis.Remote.Razor.RemoteFoldingRangeServiceFactory" />
```

Comment:
Nothing for this PR, but it does seem like a lot of overhead that we have a special servicehub service for every endpoint, rather than like a single broker service.

Or is that what service hub is essentially doing under the hood, a single pipe that gets dispatched to services as registered?

Reply:
I'll be honest, I'm not sure about the perf trade-offs of one service that does everything, versus multiple. Yes I believe service hub is essentially doing this for us, but obviously there is overhead in creating the actual services etc. I'm mostly just copying the Roslyn model of service-per-feature. The overhead is a little annoying though, I'll admit.